packages/rs-dapi-client/src/address_list.rs

```diff
@@ -47,6 +47,12 @@ pub struct AddressList {
     base_ban_period: time::Duration,
 }
 
+impl Default for AddressList {
+    fn default() -> Self {
+        Self::new()
```

Comment:
I'm not sure this is a correct address list. We should not allow creation of incorrect objects.

Reply:
You mean we should not allow an empty address list?

Reply:
No, I see it as a Builder pattern and it's ok.

It's more about `Default` trait. Here, we create empty list by default, while I think it should point to either a testnet or mainnet. But I'm not 100% sure here, so feel free to resolve this thread if you disagree.

Reply:
I added `Default` implementation because Clippy wants it if `::new` receives no args, I don't have any other idea behind this